app/preprints/-components/submit/preprint-state-machine/component.ts

```diff
@@ -130,6 +130,12 @@ export default class PreprintStateMachine extends Component<StateMachineArgs>{
         if (this.preprint.hasCoi === false) {
             this.preprint.conflictOfInterestStatement = 'null';
         }
+        if (this.preprint.hasDataLinks === PreprintDataLinksEnum.NOT_APPLICABLE) {
+            this.preprint.whyNoData = 'null';
+        }
+        if (this.preprint.hasPreregLinks === PreprintPreregLinksEnum.NOT_APPLICABLE) {
+            this.preprint.whyNoPrereg = 'null';
```

Comment:
Just noticed that these are being set to a string `'null'`. I think these should be `null`. We may need to change the Preprint model to be `whyNoPrereg!: string | null;`, and the same for coiStatement, and whyNoData to make TypeScript happy